.eslintrc.json

```diff
@@ -36,7 +36,8 @@
     "linebreak-style": "off",
     "react/prop-types": "off",
     "import/extensions": "off",
-    "react/no-array-index-key" : "off"
+    "react/no-array-index-key" : "off",
+    "no-console":"off"
   },
```

Comment:
- [ ] Please, don't tamper with the Eslint file. The rules stated in the file should not be tampered with because they help us write clean, readable and standard codes. Kindly implement this change. 👍

Reply:
✅